src/pages/workspace/distanceRates/PolicyDistanceRatesPage.tsx

```diff
@@ -0,0 +1,282 @@
+                                src={Expensicons.ArrowRight}
+                                fill={theme.icon}
+                            />
+                        </View>
+                    </View>
+                ),
+            })),
+        [customUnit?.attributes?.unit, customUnitRates, selectedDistanceRates, styles.alignSelfCenter, styles.flexRow, styles.p1, styles.pl2, styles.textSupporting, theme.icon, translate],
+    );
+
+    const addRate = () => {
+        // Navigation.navigate(ROUTES.WORKSPACE_CREATE_DISTANCE_RATE.getRoute(route.params.policyID));
+    };
+
+    const openSettings = () => {
+        // Navigation.navigate(ROUTES.WORKSPACE_DISTANCE_RATES_SETTINGS.getRoute(route.params.policyID));
+    };
+
+    const editRate = () => {
+        // Navigation.navigate(ROUTES.WORKSPACE_EDIT_DISTANCE_RATE.getRoute(route.params.policyID, rateID));
+    };
+
+    const disableRates = () => {
+        if (selectedDistanceRates.length !== customUnitRates.length) {
+            // run enableWorkspaceDistanceRates for all selected rows
+        }
+
+        setIsWarningModalVisible(true);
+    };
+
+    const enableRates = () => {
+        if (selectedDistanceRates.length !== customUnitRates.length) {
+            // run enableWorkspaceDistanceRates for all selected rows
+        }
+
+        setIsWarningModalVisible(true);
+    };
+
+    const deleteRates = () => {
+        if (selectedDistanceRates.length !== customUnitRates.length) {
+            // run deleteWorkspaceDistanceRates for all selected rows
+        }
+
+        setIsWarningModalVisible(true);
+    };
+
+    const toggleRate = (rate: RateForList) => {
+        if (selectedDistanceRates.find((selectedRate) => selectedRate.customUnitRateID === rate.value) !== undefined) {
+            setSelectedDistanceRates((prev) => prev.filter((selectedRate) => selectedRate.customUnitRateID !== rate.value));
+        } else {
+            setSelectedDistanceRates((prev) => [...prev, customUnitRates[rate.value]]);
+        }
+    };
+
+    const toggleAllRates = () => {
+        if (selectedDistanceRates.length === Object.values(customUnitRates).length) {
+            setSelectedDistanceRates([]);
+        } else {
+            setSelectedDistanceRates([...Object.values(customUnitRates)]);
+        }
+    };
+
+    const getCustomListHeader = () => (
+        <View style={[styles.flex1, styles.flexRow, styles.justifyContentBetween, styles.pl3, styles.pr9]}>
+            <Text style={styles.searchInputStyle}>{translate('workspace.distanceRates.rate')}</Text>
+            <Text style={[styles.searchInputStyle, styles.textAlignCenter]}>{translate('statusPage.status')}</Text>
+        </View>
+    );
+
+    const getBulkActionsButtonOptions = () => {
+        const options: Array<DropdownOption<WorkspaceDistanceRatesBulkActionType>> = [
+            {
+                text: translate(`workspace.distanceRates.${selectedDistanceRates.length <= 1 ? 'deleteRate' : 'deleteRates'}`),
```

Comment:
NAB we can do this in a follow up. I think a better way would be to pass a count param and do this in en.ts/es.ts, [example](https://github.com/Expensify/App/blob/18d88eff913698bbac32889fa20e3cdb286b4079/src/languages/en.ts#L610-L613). I think we can do that in a follow up as polish though, so we don't take time away from developing the core functionality now